lib/ae_page_objects/rails/application_router.rb

```diff
@@ -112,6 +112,8 @@ def normalize_url(url)
           ActionDispatch::Journey::Router::Utils.normalize_path(url) unless url =~ %r{://}
         end
       end
+
+      class Rails5 < Rails4; end
```

Comment:
Perhaps instead of subclassing, we could rename this class to reflect that it is for Rails 4+?